### DISCUSSION TITLE
How to get list of tabs in main fn of kitten.

### DISCUSSION THREAD

**Discussion by navneetankur** [kovidgoyal/kitty#7919] [Q&A] [answered]
As I understand `main` of kitten is to show a terminal ui. But the `main` doesn't seem to have access to things like `Boss`. How do I get any information about kitty in main and then show (FZF) ui based on that.

**Comment by kovidgoyal** [comment, marked as answer]
Use remote control, specifically kitten @ ls

**Reply by navneetankur** [reply]
Thanks.
I thought there would be another way.
Starting another process to get information, when kitten is already running in kitty context, felt weird. :)

**Comment by kovidgoyal** [comment]
It's a separate process, there is no shared context, other than a socket
or pipe, which is what remote control uses.

**Reply by navneetankur** [reply]
Ok. Makes sense then.
Thanks again. Kitty is great.
